src/pages/workspaces/workspace/WorkspaceFiles.ts

```diff
@@ -0,0 +1,45 @@
+import _ from 'lodash/fp'
```

Comment:
Can you name this file just "Files.ts"? The workspace part is already well implied by the directories, and it will help with CODEOWNERS as the workspaces team won't get flagged on changes to this file.